Makefile

```diff
@@ -1,5 +1,6 @@
 PREFIX ?= /usr/local
 SHARE_PREFIX ?= $(PREFIX)/share
+MAN_PREFIX ?= $(PREFIX)/man
```

Comment:
The old default destination was `$(SHARE_PREFIX)/man`, but now it's `$(PREFIX)/man`.

Reply:
@michalmuskala  fixed & pushed, sorry.

I got this 100% exactly the wrong way around, this is what I'm aiming for downstream BTW - https://www.freebsd.org/cgi/man.cgi?query=hier&apropos=0&sektion=0&manpath=FreeBSD+11.2-RELEASE+and+Ports&arch=default&format=html

> local/	  local executables, libraries, etc.  Also used as the default destination for the ports(7) framework.  Within local/, the general layout sketched out by hier for /usr should be used. Exceptions are the man directory (directly under local/ rather than under local/share/),